contracts/utils/ex00_base.cairo

```diff
@@ -37,38 +42,52 @@ end
 #
 
 @view
-func tderc20_address{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (_tderc20_address: felt):
+func tderc20_address{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (
+    _tderc20_address : felt
+):
     let (_tderc20_address) = tderc20_address_storage.read()
     return (_tderc20_address)
 end
 
 @view
-func players_registry{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (_players_registry: felt):
+func players_registry{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (
+    _players_registry : felt
+):
     let (_players_registry) = players_registry_storage.read()
     return (_players_registry)
 end
 
 @view
-func workshop_id{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (_workshop_id: felt):
+func workshop_id{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (
+    _workshop_id : felt
+):
     let (_workshop_id) = workshop_id_storage.read()
     return (_workshop_id)
 end
 
 @view
-func exercise_id{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (_exercise_id: felt):
+func exercise_id{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}() -> (
+    _exercise_id : felt
+):
     let (_exercise_id) = exercise_id_storage.read()
     return (_exercise_id)
 end
 
-
 @view
-func has_validated_exercise{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(account: felt) -> (has_validated_exercice: felt):
+func has_validated_exercise{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    account : felt
+) -> (has_validated_exercice : felt):
     # reading player registry
-	let (_players_registry) = players_registry_storage.read()
-	let (_workshop_id) = workshop_id_storage.read()
-	let (_exercise_id) = exercise_id_storage.read()
-	# Checking if the user already validated this exercice
-	let (has_current_user_validated_exercice) = Iplayers_registry.has_validated_exercice(contract_address=_players_registry, account=account, workshop=_workshop_id, exercise = _exercise_id)
```

Comment:
Should use OZ and cairo-lang.org standard of CamelCase Interface name:
`IPlayersRegistry`